lib/mongo/protocol.ex

```diff
@@ -138,24 +139,8 @@ defmodule Mongo.Protocol do
     end
   end
 
-  def handle_info({:tcp, data}, s) do
-    err = Mongo.Error.exception(message: "unexpected async recv: #{inspect data}")
-    {:disconnect, err, s}
-  end
```

Comment:
I haven't looked into the changest, but are these messages no longer being sent?

Reply:
seems latest dbconnect doesn't have the `handle_info` callback method, so I just deleted it, but I am not sure this is used by dbconnect or not